server/service/src/shipping_method/mod.rs

```diff
@@ -0,0 +1,20 @@
+use repository::shipping_method::{ShippingMethod, ShippingMethodFilter};
+
```

Comment:
Nit: I would remove all the white space between the imports... no one really pays attention to them so don't need to separate them like this 👀